src/v/cluster/controller_backend.cc

```diff
@@ -389,23 +412,29 @@ controller_backend::execute_partitition_op(const topic_table::delta& delta) {
     case op_t::del:
         return delete_partition(delta.ntp, rev);
     case op_t::update:
-        return process_partition_update(delta.ntp, delta.new_assignment, rev);
+        vassert(
+          delta.previous_assignment,
+          "update delta must have previous assignment");
```

Comment:
Please add more context here, print `*this`  And everything else. This is not enough context to debug a crash.

Reply:
@mmaslankaprv ack